packages/kit/src/views/ScanQrCode/components/index.tsx

```
@@ -146,79 +146,79 @@ export function ScanQrCode({
-            if (platformEnv.isRuntimeBrave) {
-              openUrlExternal(
-                'https://support.brave.com/hc/en-us/articles/360018205431',
-              );
-              return;
-            }
-          }
-          openSettings('camera');
-        }
-      },
-    });
+    // if (status === PermissionStatus.GRANTED) {
```

ESLint

[error] Delete `⏎`

(prettier/prettier)